helpers/filterSeats.js

```diff
@@ -58,12 +60,35 @@ module.exports = function(dust) {
       });
     }
 
+    // use Transformer show config to add Gold, Silver or Bronze to packageRate
+    function assignColoursToBands(packageRate) {
+      // assign gold, silver, bronze to packages depending on their current priceBand
+      _.forEach(bandColours, function(sectionValue, sectionKey) {
+        // check for match between transformer and existing data
+        // check for section that might not have full title ( e.g. Grand Circle in Grand Circle (Left) )
+        if (ticketRates[packageRate.ids].section === sectionKey || ticketRates[packageRate.ids].section.indexOf( sectionKey ) > -1 ) {
```

Comment:
Actually that was Tom's work that I had to merge before working on mine. But yes I think I can refactor that.